static/skin/widget.js

```diff
@@ -1 +1,87 @@
-// To be updated
\ No newline at end of file
+function disableSearchFilters(widgetStyles) {
+    const hideNavRule = `
+    .kiwixNav {
+        display: none;
+    }`;
+    const hideResultsLabelRule = `
+    .kiwixHomeBody__results {
+        display: none;
+    }`;
+    const hideTagFilterRule = `
+    .book__tags {
+        pointer-events: none;
+    }`;
+    insertNewCssRules(widgetStyles, [hideNavRule, hideResultsLabelRule, hideTagFilterRule]);
+}
```

Comment:
I think that if you introduce custom CSS handling first, then `disableSearchFilters()`, `disableDownload()`, `disableDescription()` and `hideFooter()` can be rewritten via `addCustomCss()` in a simpler way.